conf/.htaccess.uploads.fast404

```diff
@@ -2,8 +2,9 @@
 # WordPress "File not found" page rendering.
 <IfModule mod_rewrite.c>
   RewriteEngine On
-  RewriteCond %{REQUEST_FILENAME} !-f
-  RewriteCond %{REQUEST_FILENAME} !-d
-  RewriteCond %{REQUEST_FILENAME} !-l
-  RewriteRule ^.*$ - [nocase,R=404,L]
+  RewriteCond %{REQUEST_URI} ^(/wp-content/uploads).+$
```

Comment:
Major: This hardcodes the path `wp-content/uploads`, but the path can be configured / different per site.